Client/src/Pages/Monitors/CreateMonitor/index.jsx

```diff
@@ -40,6 +42,21 @@ const CreateMonitor = () => {
   const [https, setHttps] = useState(true);
   const [errors, setErrors] = useState({});
 
+  useEffect(() => {
+    if(monitorId){
+      const data = monitors.find((monitor) => monitor._id === monitorId);
+      if (!data) {
+        console.error("Error fetching monitor of id: " + monitorId);
+        navigate("/not-found");
```

Comment:
Please add the `{replace: true}` config object to `navigate` here so we don't get stuck in a loop if the user presses back form the 404 page.